src/content/post/2025-10-17-4941c349.md

```
@@ -0,0 +1,28 @@
+[Nix meetup #4](https://nix-ja.connpass.com/event/369476/)に参加した。
+
+資料は以下にデプロイしている。
+
+[チームで nix を使うために考えていること（仮） | ドクセル](https://www.docswell.com/s/Omochice/KDW43N-2025-10-19-nix-meetup-4)
+
+端的に言うとnixをチームに導入するなら初手は[jetify-com/devbox](https://github.com/jetify-com/devbox)がいいんじゃないのか、という内容。
+
+要旨は以下の通り。
+
+- 大体の人がビルドの冪等性に興味がない
+    - 加えて新しい言語を本人のモチベーションなしでやらせるのは大変
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)